### PR TITLE
Fix: No content in readme

### LINKED ISSUE #35: No content in readme
Here's the link to our documentation. You will find the link to our discord community server in there incase you need to ask any questions: https://code-collabo.gitbook.io/docs/collabo-docs-home/readme
#

#### Issue description
There is no descriptive content in the Collabo Web App repo: https://github.com/code-collabo/collabo-web-app

#### Additional context
Take a clue from the following repos to add a simplified readme content for the Collabo Web App repo:
- https://github.com/code-collabo/node-mongo-scripts
- https://github.com/code-collabo/docs

#### Testing checklist
Note: Add the testing checklist below (without this note) to your pull request when submitting your fix. This will guide (you and) the contributors who will review your fix to know what is important to check or test for.
- [ ] Simplified readme content added for Collabo Web App repo, following the footsteps of the example repos.
- [ ] I certify that I ran my checklist

### PR DESCRIPTION
**Pull request by aybeedee**
**This pull request makes the following changes**
* Fixes code-collabo/docs#35

**General checklist**
- [X] File or folder now contains changes as specified in the issue i worked on
- [X] I have linked the issue I worked on to this pull request submitted by me

**Testing checklist**

- [X] Simplified readme content added for Collabo Web App repo, following the footsteps of the example repos.
- [X] I certify that I ran my checklist

Ping @code-collabo/web-app

### CODE REVIEW

**Comment by Ifycode** [comment]
**Review comment 1:** @aybeedee Awesome that you could link the issue fixed to the pull request. But help change your pull request title to "Fix: No content in readme". You can read more about this on the contributing guide page of our docs: https://code-collabo.gitbook.io/collabo-contributor/collabo-contributor-docs/contributing-guide

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
 # Collabo Web App
-Content coming soon
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://code-collabo.gitbook.io/collabo-contributor/collabo-web-app-project/contribute-to-collabo-web-app) [![GitHub issues](https://img.shields.io/github/issues/code-collabo/collabo-web-app?color=red)](https://github.com/code-collabo/collabo-web-app/issues) [![GitHub pull requests](https://img.shields.io/github/issues-pr/code-collabo/collabo-web-app?color=goldenrod)](https://github.com/code-collabo/collabo-web-app/pulls)
+
```
Comment (Ifycode) [inline README.md:4]: **Review comment 2:** Badges nicely added. You can remove the issues badge for now. We have a different way of accessing issues according to our docs.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
 # Collabo Web App
-Content coming soon
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```
Comment (Ifycode) [inline README.md]: **Review comment 3:** We have not set up @all-contributors bot for this repo, so you can remove the all contributors badge and comments for now. This part is addressed in another issue ticket.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,25 @@
...
+
+To learn more about Code Collabo's community and other projects, feel free to visit the official [Code Collabo Documentation](https://code-collabo.gitbook.io/docs)
+
+## Contributors
+
+Thanks to these amazing contributors to the collabo web application project. Contributions of any kind welcome!
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<a href="https://github.com/code-collabo/collabo-web-app/graphs/contributors"><img src="https://contrib.rocks/image?repo=code-collabo/collabo-web-app" alt="All the amazing contributors of the Collabo Web App"></a>
+<!-- ALL-CONTRIBUTORS-LIST:END -->
```
Comment (Ifycode) [inline README.md]: **Review comment 4:** We have not set up @all-contributors bot for this repo, so you can remove the entire contributors section for now. This part is addressed in another issue ticket.

**Review by Ifycode — CHANGES_REQUESTED** [review]

**Comment by aybeedee** [comment]
Made all the required review changes. Also, I have a couple of questions. Why was the README from the "unreviewed" branch dropped? And also what is meant by "Ping @code-collabo/web-app"/how is it supposed to work (it's part of the PR Template).

**Comment by Ifycode** [comment]
Good question @aybeedee 
We are centralizing our docs so that we don't have to repeat information. You can take a look at the contributing doc for the Collabo web app to see: https://code-collabo.gitbook.io/collabo-contributor/collabo-web-app-project/contribute-to-collabo-web-app
So now, we just send people to the docs from the readme for all the info they need.
#
Just ignore the ping part of the PR template for now. It may just be one of the clean ups we will do.

**Review by Ifycode — APPROVED** [review]
Neatly done @aybeedee

**Pull request merged**